File: C#/01 - sumUptoNTerms.cs
using System;

public class sumUptoNTerms
{
    public static void Main(string[] args)
    {
        Console.Write("Enter number of numbers to add: ");
        int n = Convert.ToInt32(Console.ReadLine());
        int sum = 0;
        for (int i = 0; i < n; i++)
        {
            sum += i;
        }
        Console.WriteLine("The sum is " + sum);
    }
}

File: C/01 - sumUptoNTerms.c
#include <stdio.h>

int main() {
    int n;
    int sum = 0;
    printf("Enter number of top n numbers to add: ");
    scanf("%d", &n);
    for (int i = 0; i < n; i++){
        sum += i;
    }
    printf("The sum is %d", sum);
    return 0;
}